@here/harp-mapview/lib/MapView.ts

```diff
@@ -1335,9 +1335,7 @@ export class MapView extends THREE.EventDispatcher {
     addDataSource(dataSource: DataSource): Promise<void> {
         const twinDataSource = this.getDataSourceByName(dataSource.name);
         if (twinDataSource !== undefined) {
-            throw new Error(
-                `A DataSource with the name "${dataSource.name}" already exists in this MapView.`
-            );
+            return new Promise(resolve => resolve());
```

Comment:
Isn't change discussed in #424 ? I would remove it from this patch because this as already discussed this approach is broken.